src/integrations/prefect-docker/prefect_docker/deployments/steps.py

```diff
@@ -151,6 +179,10 @@ def build_docker_image(
                 platform: amd64
         ```
     """  # noqa
+
+    if ignore_cache:
+        logger.debug("Ignoring `@cacheable` decorator for build_docker_image.")
```

Comment:
This log message should probably be moved into `cacheable` since the function isn't aware of its decorators.